File: muretWebFrontent/src/app/model/SVGSet.ts
import {AgnosticTypeSVGPath} from './agnostic-type-svgpath';

export class SVGSet {
  x: number;
  y: number;
  em: number;
  paths: Array<AgnosticTypeSVGPath>;


  constructor(x: number, y: number, em: number, paths: Array<AgnosticTypeSVGPath>) {
    this.x = x;
    this.y = y;
    this.em = em;
    this.paths = paths;
  }
}

File: im3ws/src/main/java/es/ua/dlsi/grfia/im3ws/muret/entity/SVGSet.java
package es.ua.dlsi.grfia.im3ws.muret.entity;

import java.util.List;
import java.util.Map;

public class SVGSet {
    double x;
    double y;
    double em;

    /**
     * key = AgnosticTypeString
     * value = SVG d param of SVG path element
     */
    List<AgnosticTypeSVGPath> paths;

    public SVGSet(double x, double y, double em, List<AgnosticTypeSVGPath> paths) {
        this.x = x;
        this.y = y;
        this.em = em;
        this.paths = paths;
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }

    public double getEm() {
        return em;
    }

    public List<AgnosticTypeSVGPath> getPaths() {
        return paths;
    }
}
